4-governance/dubbo-samples-cat/case-configuration.yml

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+services:
+  zookeeper:
+    image: zookeeper:latest
+
+  cat-provider:
+    type: app
+    basedir: dubbo-samples-cat-provider
+    mainClass: org.apache.dubbo.samples.cat.provider.CatProvider
+
+  cat-consumer:
+    type: test
+    basedir: dubbo-samples-cat-consumer
+    tests:
+      - "**/*IT.class"
+    systemProps:
+      - zookeeper.address=cat-provider
+      - zookeeper.port=2181
+      - dubbo.address=cat-provider
+      - dubbo.port=20880
```

Comment:
```suggestion
    systemProps:
      - zookeeper.address=cat-provider
      - zookeeper.port=2181
      - dubbo.address=cat-provider
      - dubbo.port=20880
      - cat.host=cat-consumer
```

Reply:
It seems that it can't connect to zookeeper